src/core/issues-notes.js

```diff
@@ -146,6 +168,35 @@ function handleIssues($ins, ghIssues, conf) {
   }
 }
 
+//derives the text-color-class based on the illumination score of the labelColor
+function deriveTextColorClass(hexColor) {
+  const rgb = parseInt(hexColor, 16);
+  if (isNaN(rgb)) {
+    throw new TypeError(`Invalid hex color: ${hexColor}`);
```

Comment:
Thanks for explaining this

Reply:
No problem. Trust me, you'll be thanking me for years :) Keep an eye out for that from other developers. It can make code super difficult to debug, specially async code.